### ISSUE TITLE
Nil access in appeal#from_records due to unknown VSO

### ISSUE THREAD

**Issue by orischwartz** [department-of-veterans-affairs/caseflow-certification#246]
```
NoMethodError (undefined method `[]' for nil:NilClass):
  app/models/appeal.rb:153:in `from_records'
  app/services/appeal_repository.rb:25:in `find'
  app/models/appeal.rb:103:in `find'
  app/controllers/certifications_controller.rb:95:in `appeal'
  app/controllers/certifications_controller.rb:75:in `verify_access'
```

**Comment by orischwartz** [comment]
NSD ticket I8307241FY16

**Comment by orischwartz** [comment]
Closed by #245
